src/components/TokenSelect/CommonBases.tsx

```diff
@@ -0,0 +1,66 @@
+import { Currency } from '@uniswap/sdk-core'
+import Row from 'components/Row'
+import TokenImg from 'components/TokenImg'
+import { BASES_TO_CHECK_TRADES_AGAINST } from 'constants/routing'
+import { transparentize } from 'polished'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+import { currencyId } from 'utils/currencyId'
+
+const BasesContainer = styled(Row)`
+  margin: 0 1.25em;
+`
+
+const BaseWrapper = styled.div<{ active?: boolean }>`
+  align-items: center;
+  background-color: ${({ theme, active }) => (active ? transparentize(0.76, theme.active) : 'transparent')};
+  border: 1px solid ${({ theme, active }) => (active ? theme.active : theme.outline)};
+  border-radius: 1em;
+  color: ${({ theme, active }) => (active ? theme.active : theme.primary)};
+
+  display: flex;
+  :hover {
```

Comment:
I think this should be `:hover, :focus` so that tabbing into an element (where the 'Enter' keypress is applicable) also shows the same UI.